utils/formatTable.ts

```diff
@@ -18,6 +18,7 @@ type DataType = {
 }
 
 type TableDataType = {
+  sort_key: string
```

Comment:
This should be replaced with camelcase "sortKey" to pass the eslint rule.

Reply:
That's right! Thank you for your comment.

Reply:
Reviewed and ready to be merged.